java/clients/src/test/java/sleeper/clients/teardown/ShutdownSystemProcessesIT.java

```diff
@@ -158,7 +172,8 @@ void shouldStopECSTaskWhenOneIsFound() throws Exception {
                 .willReturn(aResponse().withStatus(200)));
         stubFor(listActiveClustersRequest()
                 .willReturn(aResponseWithNumRunningClusters(0)));
-
+        stubFor(listActiveApplicationsRequest()
+                .willReturn(aResponseWithNumRunningApplications(0)));
         // When
```

Comment:
This is missing an empty line before the When.

Reply:
Added